### DISCUSSION TITLE
Blender version support

### DISCUSSION THREAD

**Discussion by ros-dorian** [mitsuba-renderer/mitsuba-blender#80] [General]
Hello @bathal1, I see that there have been a few issues regarding CI mostly due to the different versions of Mitsuba/Blender that have been coming out.

In an effort to make CI green again, what would be a good rule to determine which version of Blender should be supported (e.g., all LTS versions of Blender or the newest one) ?

Moreover, the current test pipeline seems a bit flaky. Should some effort be put into making it more reliable or could we consider temporarily removing some of the tests ?

Cheers!

**Comment by bathal1** [comment]
Hi @ros-dorian ! 
We have been working on getting the CI up and running again, most issues should be fixed on the `test-fixes` branch (see #77). The last remaining issue is a DLL loading conflict caused by Blender moving from static to dynamic libraries, causing a clash between some dependencies of blender and mitsuba. The fix is already pushed to mitsuba's master, but we'll have to wait for the next release to integrate it in the addon.

Regarding supported versions, I think it would be sufficient to target LTS versions. Right now that would be 3.3 and 3.6. Both should be supported, though we are waiting for the CI fixes to be merged before making a new release properly supporting the changes in the mesh layout in 3.6.
